File: Keywords/Keywords.cs
using System;

namespace Keywords
{
    class Parent
    {
        public string name = "Parent";
        
        public Parent()
        {
            Console.WriteLine("Parent Constructor");
        }
    }

    sealed class Child : Parent // 'sealed' prevents inheritance from Child
    {
        public string name = "Child";
        public static int count = 0; // Static variable

        public Child() : base() // Call Parent constructor
        {
            count++;
            Console.WriteLine("Child Constructor");
        }

        public void Display()
        {
            // 'this' refers to current class attribute
            Console.WriteLine("Current Name: " + this.name);
            
            // 'base' refers to parent class attribute
            Console.WriteLine("Parent Name: " + base.name);
            
            Console.WriteLine("Total Children Created: " + count);
        }
    }

    class Program
    {
        static void Main(string[] args)
        {
            Child c1 = new Child();
            c1.Display();

            Child c2 = new Child();
            c2.Display();

            // Accessing static variable via Class
            Console.WriteLine("Final Count: " + Child.count);
        }
    }
}

File: Keywords/Keywords.java
package Keywords;

class Parent {
    String name = "Parent";

    Parent() {
        System.out.println("Parent Constructor");
    }
}

class Child extends Parent {
    String name = "Child";
    static int count = 0; // Static variable
    final int MAX_LIMIT = 100; // Final (Constant)

    Child() {
        super(); // Call Parent constructor
        count++; // Increment static counter
        System.out.println("Child Constructor");
    }

    void display() {
        // 'this' refers to current class attribute
        System.out.println("Current Name: " + this.name);

        // 'super' refers to parent class attribute
        System.out.println("Parent Name: " + super.name);

        System.out.println("Total Children Created: " + count);
    }
}

public class Keywords {
    public static void main(String[] args) {
        Child c1 = new Child();
        c1.display();

        Child c2 = new Child();
        c2.display();

        // Accessing static variable directly via Class
        System.out.println("Final Count: " + Child.count);
    }
}
